### DISCUSSION TITLE
"Show in Explorer" option not working

### DISCUSSION THREAD

**Discussion by joncode21** [phcode-dev/phoenix#1787] [General]
When i try to right click  a folder and  select "show in explorer", it does not direct me to the direct folder, 

instead it just directs me to a default windows explorer page

**Comment by charlypa** [comment]
@joncode21 
Are you trying to open live preview in windows explorer

**Reply by joncode21** [reply]
Nope. 

It is when I want to view a folder in the file explorer page by going to the IDE and right clicking on the folder  and selecting "Show in Explorer".

It does not auto redirect me to it.

**Comment by abose** [comment]
@joncode21 which windows version is this.

**Reply by joncode21** [reply]
Release 3.8 build 3.8.8-20449
build timestamp: 2024-06-27T08:37:39.564Z

**Reply by abose** [reply]
Windows 10/11 version

**Comment by joncode21** [comment]
ok I forgot to reply here. The "Show in explorer"  button works for files but not for folders. Please help to update it for folders too.
